content/av-content.md

```diff
@@ -133,7 +138,9 @@ For other guidance including recording, see another page of this resource: [Sign
 
 ### Plan for audio description of visual information – _storyboarding, recording_  (WCAG A, AA) {#plan-description}
 
-_Description_ provides content to people who are blind and others who cannot see the video adequately. It describes the visual information needed to understand the content.
+_Description_ provides content to people who are blind and others who cannot see the video adequately. It describes the visual information needed to understand the content. This usually includes all text in videos. 
```

Comment:
Minor suggestion: people who are blind, have low vision, or otherwise who cannot see the video adequately.
Rationale: not only blind people are impacted, and I think that reminding about low vision would be useful here.

Reply:
done

Reply:
Thanks